Project.toml

```diff
@@ -41,7 +41,7 @@ HDF5 = "0.14, 0.15, 0.16"
 IsApprox = "0.1"
 KrylovKit = "0.4.2, 0.5, 0.6"
 LinearMaps = "3"
-NDTensors = "0.1.50"
+NDTensors = "0.2.0"
```

Comment:
Can we use `"0.1.50, 0.2"`? Otherwise, ITensorGPU won't be able to access updated versions of ITensors.jl.

Reply:
Yes, this works and is updated.